spec/integration_spec.rb

```diff
@@ -25,7 +25,8 @@ def build_organizer(options = {}, &block)
   #  └─ interactor5
 
   let(:organizer) {
-    build_organizer(organize: [organizer2, interactor3, organizer4, interactor5]) do
+    interactors = [organizer2, interactor3, organizer4, interactor5]
+    build_organizer(organize: interactors) do
```

Comment:
Yep. Both ways have their benefits and downsides. Introducing intermediate variable seems a bit clumsy, but it keeps lines short. Short lines are easier to perceive and they are two-panes-friendly:
![2017-03-22_15-36-32](https://cloud.githubusercontent.com/assets/3628124/24198083/5e966b4c-0f15-11e7-908b-d9ac070a9354.png)